src/main/resources/css/language.css

```
@@ -85,18 +87,24 @@ and (min-device-width: 320px)
         width: 100vw;
         height: 79px;
         margin: 0;
-        border: solid 1px #111A3133;
+        border: none;
+        border-top: 1px solid #111A3133 !important;
         border-radius:0;
     }
+    .l-list-container > div:last-child a.def-card {
+        border-bottom: 1px solid #111A3133;
+    }
```

we should be able to move this to common.css. 
```suggestion
    .l-list-container > div:last-child a {
     border-bottom: 1px solid #111A3133;
    }
```
we can leave it general and not have to specify def-card or chaptercard after the `a`